apps/mull-ui/src/app/pages/waste-recognition-page/identified-waste-modal.tsx

```diff
@@ -0,0 +1,55 @@
+import { faTimes } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { Dialog } from '@material-ui/core';
+import React from 'react';
+import { svgMap } from '../../../utilities';
+import './identified-waste-page.scss';
+
+export interface IdentifiedWasteModalProps {
+  wasteIcon?: string;
+  wasteTitle?: string;
+  wastePicture?: string;
+  wasteInfo?: string;
+}
+
+export const IdentifiedWasteModal = ({
+  wasteIcon,
+  wasteTitle,
+  wastePicture,
+  wasteInfo,
+}: IdentifiedWasteModalProps) => {
+  wasteIcon = '../../.' + svgMap[1];
+  wasteTitle = 'This is recyclable!';
+  wastePicture = 'http://pm1.narvii.com/5951/2922186bdf76edeb36250994eb99f1c32f31aea9_00.jpg';
+  wasteInfo = 'Needs to be cleaned before recycling. Otherwise it goes to trashbin.';
+
+  const [open, setOpen] = React.useState(false);
+  const handleOpen = () => {
+    setOpen(true);
+  };
+  const handleClose = () => {
+    setOpen(false);
+  };
```

Comment:
💡 Honestly, if your methods are only calling setOpen, you might as well remove them and call setOpen directly in your onClicks